reference/UiTPAS.v2.json

```diff
@@ -334,150 +352,955 @@
+        }
+      ],
+      "get": {
+        "summary": "Get organizer distribution keys",
+        "tags": [
+          "Organizers"
         ],
-        "x-examples": {
-          "Example": {
-            "id": 21345,
-            "tariff": {
-              "id": "COUPON1234",
-              "name": "Cultuurbon 6 euro",
-              "price": 1.5
-            },
-            "uitpasNumber": "0560002524314",
-            "eventId": "31e926e2-a35f-11eb-bcbc-0242ac130002"
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/components/schemas/DistributionKey"
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Not Found",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Error"
+                }
+              }
+            }
           }
         },
-        "description": "The TicketSaleRegistration is a response object of the POST /ticketSales request. It can be used to provide the user with an overview or to identify the ticket sales in case they need to be canceled.",
-        "x-internal": false,
-        "x-tags": [
-          "Model"
-        ]
-      },
-      "TicketSaleRequest": {
-        "title": "TicketSaleRequest",
-        "type": "object",
-        "description": "The TicketSaleRequest is a request object used to register ticket sales in POST /ticketSales.",
-        "x-examples": {
-          "Example ticket sale request": {
-            "uitpasNumber": "0560002524314",
-            "tariffId": "COUPON1234",
-            "eventId": "31e926e2-a35f-11eb-bcbc-0242ac130002",
-            "regularPrice": 10,
-            "regularPriceLabel": "Optional descriptive label of the regular price. Used in financial reporting."
+        "operationId": "get-organizer-distribution-keys",
+        "security": [
+          {
+            "User Acces Token (see Authentication docs)": []
           }
-        },
-        "properties": {
-          "uitpasNumber": {
-            "type": "string"
-          },
-          "tariffId": {
-            "type": "string"
-          },
-          "eventId": {
+        ],
+        "description": "Get manual distribution keys for an organizer, so it can be set to a specific event using [TODO](#todo)\n\n<!-- theme: warning -->\n\n> **This endpoint is only needed for exceptional cases.** In most cases distribution keys are set automatically on events, so you don't need to retrieve the possible distribution keys for the event's organizer."
```

Comment:
Wat moet er in plaats van `[TODO](#todo)` komen? Een link naar het endpoint om distribution keys op een event te configureren?